README.md

```diff
@@ -19,21 +49,26 @@ Once integrated, your marketing team will be able to better understand your app
 push notifications via FCM (Firebase Cloud Messaging) and APNs (Apple Push Notification Service).
 
 ## Requirements
+
```

Comment:
Just curious if an auto-formatter added a blank line after all headers or something?

Reply:
Yeah looks like opening it in intellij did this.

Reply:
No problem, just curious.